### PR TITLE
JSHint fixes

### PR DESCRIPTION
**Pull request by shmert**
Using single-quotes

'use strict'

flag d3 as global var

### CODE REVIEW

**Comment by shmert** [comment]
Hi, just trying out the library, fixed some complaints that JSHint was having.

File: gantt-chart-d3.js [hunk 1]
```diff
@@ -1,221 +1,256 @@
...
  * @version 2.1
  */
 
-d3.gantt = function() {
-    var FIT_TIME_DOMAIN_MODE = "fit";
-    var FIXED_TIME_DOMAIN_MODE = "fixed";
-    
+d3.gantt = function () {
+    var FIT_TIME_DOMAIN_MODE = 'fit';
+    //var FIXED_TIME_DOMAIN_MODE = 'fixed';
```
Comment (dk8996) [inline gantt-chart-d3.js]: I noticed you removed this?

**Comment by dk8996** [comment]
I am cool with this.  You commented out the following:
//var FIXED_TIME_DOMAIN_MODE = 'fixed';

**Comment by dk8996** [comment]
@shmert take a look.

**Pull request closed without merging**